jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/SkaffoldFilesOutput.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.plugins.common;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Builds a JSON string containing files and directories that <a
+ * href="https://github.com/GoogleContainerTools/skaffold">Skaffold</a> should watch for changes
+ * (and consequently trigger rebuilds).
+ *
+ * <p>{@code build} consists of build definitions. Changes in these files/directories indicate that
+ * the project structure may have changed, so Skaffold will refresh the file watch list when this
+ * happens.
+ *
+ * <p>{@code inputs} consist of source/resource files/directories. Skaffold will trigger a rebuild
+ * when changes are detected in these files.
+ *
+ * <p>{@code ignore} consists of files/directories that the Skaffold file watcher should not watch.
+ *
+ * <p>Example:
+ *
+ * <pre>{@code
+ * {
+ *   "buildFiles": [
```

Comment:
oops, I missed this one, doc needs to be updated in "build"

Reply:
Yeah I just realized that myself, haha